features/case/case-detail/tabs/header-widget.md

```diff
@@ -0,0 +1,109 @@
+# Case Header Widget
+
+A **case header widget** displays important case information at the top of a case detail page.  
+This section remains visible at all times, regardless of which tab is selected.
+
+## Configuring a Case Header Widget
+
+The configuration for a case header widget is part of the **case definition** setup.
+
+### Steps to configure
+
+1. Go to the **`Admin`** menu.
+2. Open the **`Cases`** menu and select the case you want to configure widgets for.
+3. Open the **`Case details`** tab.
+4. Click the **`Header`** tab.
+
+On this page, you can configure **one header widget**.  
+To create it, click the **`Add widget`** button.
+
+You will then go through three setup steps:
+
+1. **Choose widget type**
+2. **Choose widget style**
+3. **Choose widget content**
+
+> Note: Case header widgets do **not** include width or condition settings.  
+> Only the **`Fields`** widget type is supported.
+
+For a detailed explanation of these steps, see the [widget configuration documentation](./widgets.md).
+
+<figure>
+  <img src="../../../../.gitbook/assets/add-case-header-widget.png" alt="Add case header widget">
+</figure>
+
+---
+
+## Access Control
+
+Access to case header widgets can be managed through **access control settings**.  
+For more details, visit the [Access Control documentation](https://docs.valtimo.nl/features/access-control).
+
+### Resources and Actions
+
+| Resource Type | Action | Effect |
+|----------------|---------|--------|
+| `com.ritense.document.domain.impl.JsonSchemaDocument` | `view` | Allows viewing a case header widget on the case detail page |
+
+---
+
+## Autodeployment via IDE
```

Comment:
Is this the default that other pages with this kind of information does? I would change it to "Autodeployment at startup".